src/meta/Focus.ts

```diff
@@ -0,0 +1,59 @@
+import { Base } from './Base';
+import { createHandle } from '@dojo/core/lang';
+import { assign } from '@dojo/core/lang';
+import global from '@dojo/shim/global';
+
+export interface FocusResults {
+	active: boolean;
+	containsFocus: boolean;
+}
+
+const defaultResults = {
+	active: false,
+	containsFocus: false
+};
+
+export class Focus extends Base {
+	private _activeElement: Element;
+	private _boundFocusHandler = this._onFocus.bind(this);
+
+	public get(key: string | number): FocusResults {
+		const node = this.getNode(key);
+
+		if (!node) {
+			return assign({}, defaultResults);
```

Comment:
Do we need to use `assign` from dojo/core here? Can we just do:
```ts
return { ...defaultResults };
```

Reply:
I think I just copied this pattern from other meta providers, but `{ ...defaultResults }` looks better